firebase-datatransport/gradle.properties

```diff
@@ -1,4 +1,3 @@
-version=18.1.9
+version=18.1.10
```

Comment:
Why is this incrementing past an unreleased patch version?

Reply:
Great catch, thanks! Fixed